superbench/benchmarks/micro_benchmarks/gpu_copy_performance/gpu_copy.cu

```diff
@@ -545,18 +652,33 @@ int main(int argc, char **argv) {
         // Scan all GPUs
         for (int j = 0; j < gpu_count; j++) {
             // Host-to-device benchmark
-            if (opts.htod_enabled) {
+            if (opts.htod_enabled || opts.htod_with_dtoh_enabled) {
                 args.is_src_dev_gpu = false;
                 args.is_dst_dev_gpu = true;
                 args.dst_gpu_id = j;
                 args.worker_gpu_id = j;
                 if (opts.sm_copy_enabled) {
                     args.is_sm_copy = true;
-                    args_list.push_back(args);
+                    if (opts.htod_enabled) {
```

Comment:
Why only consider htod_enabled?

Reply:
Making htod a bidirectional test is the same as making dtoh a bidirectional test, i.e. always dtoh + htod. So only htod part is modified here.

Reply:
Now it depends on user to set only htod instead of htod + dtoh in bidirectional mode avoid redundant tests.